guide/src/class.md

```diff
@@ -222,12 +224,12 @@ impl MyClass {
 }
 ```
 
-Getter or setter function's name is used as property name by default. There are several
-ways how to override name.
+A getter or setter's function name is used as the property name by default. There are several
+ways how to override the name.
 
-If function name starts with `get_` or `set_` for getter or setter respectively.
-Descriptor name becomes function name with prefix removed. This is useful in case of
-rust's special keywords like `type`.
+If a function name starts with `get_` or `set_` for getter or setter respectively,
+the descriptor name becomes the function name with this prefix removed. This is also useful in case of
+Rust keywords like `type` (raw identifiers can be used since Rust 2018).
```

Comment:
```suggestion
Rust keywords like `type` ([raw identifiers](https://doc.rust-lang.org/nightly/edition-guide/rust-2018/module-system/raw-identifiers.html) can be used since Rust 2018).
```

Reply:
I'll put in the non-nightly link.